source/jsonnet/england-wales/ccs/blocks/individual/other_uk_address.jsonnet

```diff
@@ -10,28 +10,13 @@ local rules = import 'rules.libsonnet';
     type: 'General',
     answers: [
       {
-        id: 'other-uk-address-answer-building',
-        label: 'Address line 1',
+        id: 'other-uk-address-answer',
         mandatory: false,
-        type: 'TextField',
-      },
-      {
-        id: 'other-uk-address-answer-street',
-        label: 'Address line 2',
-        mandatory: false,
-        type: 'TextField',
-      },
-      {
-        id: 'other-uk-address-answer-city',
-        label: 'Town or city',
-        mandatory: false,
-        type: 'TextField',
-      },
-      {
-        id: 'other-uk-address-answer-postcode',
-        label: 'Postcode',
-        mandatory: false,
-        type: 'TextField',
+        type: 'Address',
+        lookup_options: {
+          address_type: 'Residential',
+          region_code: std.extVar('region_code'),
+        },
```

Comment:
There is repetition but I not sure if it warrants a partial at the current state.

Reply:
I didn't think so..